libvast/test/table_slice.cpp

```diff
@@ -115,4 +116,66 @@ TEST(random integer slices) {
   CHECK_LESS_EQUAL(*highest, 200);
 }
 
+TEST(select all) {
+  auto sut = zeek_full_conn_log_slices.front();
+  sut.unshared().offset(100);
+  auto select_result = select(sut, make_ids({{100, 200}}));
+  REQUIRE_EQUAL(select_result.size(), 1u);
+  CHECK_EQUAL(select_result[0], sut);
+}
+
+TEST(select none) {
+  auto sut = zeek_full_conn_log_slices.front();
+  sut.unshared().offset(100);
+  auto select_result = select(sut, make_ids({{200, 300}}));
+  CHECK_EQUAL(select_result.size(), 0u);
+}
+
+TEST(select prefix) {
+  auto sut = zeek_full_conn_log_slices.front();
+  sut.unshared().offset(100);
+  auto select_result = select(sut, make_ids({{0, 150}}));
+  REQUIRE_EQUAL(select_result.size(), 1u);
+  CHECK_EQUAL(select_result[0]->rows(), 50u);
+  CHECK_EQUAL(to_events(*select_result[0]), to_events(*sut, 0, 50));
+}
+
+TEST(select off by one prefix) {
```

Comment:
```suggestion
TEST(select off-by-one prefix) {
```

Reply:
That's what I had at first. The problem is that clang-format is "too clever" here and formats this as `off - by - one`. I figured it's more readable this way. :/